package.json

```diff
@@ -39,6 +39,7 @@
     "lint:pack": "packlint sort -R",
     "lint:pub": "turbo run lint:pub",
     "postinstall": "husky install",
+    "preinstall": "corepack enable",
```

Comment:
do we need to postinstall command ? it is listened git history

Reply:
More explanation is needed

Reply:
If I know exactly right, I understand that this command is only necessary if we don't manage the `.husky` folder with git. 
We are managing husky with git, so I think it's a command we don't need.

---

I tested it in detail, and it didn't work without `.husky/_` folders, and `.husky/_` is a folder registered in gitignore, so I think I need the husky install command every time. Is it bad to commit `.husky/_` to git?